en_US/advanced/http-api.md

```diff
@@ -976,7 +976,7 @@ Normal shutdown_count*
 | normal     | Integer   | Number of normally closed connections, only returned when the count is greater than 0 |
 | kicked     | Integer   | Number of manually dropped connections, only returned if the count is greater than 0 |
 | discarded  | Integer   | Number of connections dropped because `Clean Session` or `Clean Start` is `true` |
-| takeovered | Integer   | Number of connections takeovered because `Clean Session` or `Clean Start` is `false` |
+| takenover | Integer   | Number of connections takenover because `Clean Session` or `Clean Start` is `false` |
```

Comment:
Nano nit: the second `takenover` looks like it should be `taken over`, as it's not a value name.